app/src/main/java/sergio/splashbaseviewer/mvp/events/DownloadFailureEvent.java

```diff
@@ -0,0 +1,17 @@
+package sergio.splashbaseviewer.mvp.events;
+
+/**
+ * @author s.ruiz
+ */
+
+public class DownloadFailureEvent {
+    String errorMessage;
```

Comment:
Dont modify this. But if you define the getter method, the field should be private.